Sekoia.io/sekoiaio/triggers/cases.py

```diff
@@ -18,36 +18,41 @@ class SecurityCasesTrigger(_SEKOIANotificationBaseTrigger):
         ("case", "alerts-updated"),
     ]
 
-    def _filter_notifications(self, message) -> bool:
+    def _filter_by_mode(self, message) -> bool:
         case_attrs = message.get("attributes", {})
-
-        # Filter by mode
         mode_filter = self.configuration.get("mode_filter")
         if mode_filter and case_attrs.get("manual") != (mode_filter == "manual"):
             return False
 
-        # Filter by priority UUIDs
+        return True
+
+    def _filter_by_priority(self, message) -> bool:
+        case_attrs = message.get("attributes", {})
         priority_uuids_filter = self.configuration.get("priority_uuids_filter")
         if priority_uuids_filter and case_attrs.get("custom_priority_uuid") not in priority_uuids_filter:
             return False
 
-        # Filter by assignees
+        return True
+
+    def _filter_by_assignees(self, message) -> bool:
+        case_attrs = message.get("attributes", {})
         assignees_filter = self.configuration.get("assignees_filter")
         if assignees_filter:
             case_details = self._retrieve_case_from_caseapi(case_attrs.get("uuid"))
```

Comment:
The filter must be applied on the case from the API. So, this line is useless, as you must already have fetched the case.